### ISSUE TITLE
Removal of the 'enhancement' label

### ISSUE THREAD

**Issue by paramat** [minetest/minetest#4677]
Please can we stop using this, as it's so vague and unhelpful. Almost all PRs can be considered 'enhancements' because they improve the game. Because it is vague it is mostly (correctly) not used, so now the label has even less use in filtering PRs.

**Comment by paramat** [comment]
Heh.

**Comment by Zeno-** [comment]
:+1:

**Comment by tobyplowy** [comment]
👍

**Comment by paramat** [comment]
Alternatively, we agree on a way to use it and make sure it is used, and rename it to be clearer.
We cannot, as far as i know, filter out certain labels, so if we want to filter for PRs that are not bugfixes or maintenance, a 'new feature' (or whatever) label might be useful.
We might need to have a set of categories for PRs and make sure to always use one of the categories:
bugfix
maintenance (this one's a little unclear too).
feature
code improvement
...

**Comment by nerzhul** [comment]
If you remove Enhancement label i think we should add "New feature" label, because there is no label for this atm except Enhancement

**Comment by rubenwardy** [comment]
It may be worth combining new features and feature request into "Feature"

Same for bug and bugfix

**Comment by nerzhul** [comment]
@rubenwardy absolutely no. New feature permits to view directly new features, this should not be merged with feature requested by users.
for bug and bugfix same problem, a bug is not a bugfix

**Comment by rubenwardy** [comment]
Nope, you'll be able to see based on whether the thread is an issue or a pull request

**Comment by paramat** [comment]
Nope :] i created the bugfix label myself, there's no need to squeeze both back into one when it is now clearer.
A 'new feature' or 'feature' label would be a good renaming for 'enhancement' i think.

**Comment by est31** [comment]
Enhancement != Feature. Don't do a rename but instead remove the enhancement label and add a feature label to all PRs/Issues that still are about features.

**Comment by paramat** [comment]
Sure that's fine by me. Does anyone know what enhancement means though? :]

**Comment by paramat** [comment]
Done.
